### DISCUSSION TITLE
Permission issues internaly on the docker image?

### DISCUSSION THREAD

**Discussion by mattish91** [seerr-team/seerr#952] [Q&A]
I have Jellyseerr running on Casa OS (have been working for months) BUT i had a huge disk crash a few days ago now and had do migrate my data to a new disk in order to solve this issue. How ever after moving my data dir to a new disk in Casa OS i now can't login anymore.

I most likely believe the config files got the wrong permissions after the move of the files.

Can someone please show me what group:user the docker image should have as mine all show up as root:root.

I have had this issue with multiple other docker images and after a chown user:user it all starts working again.

**Comment by mattish91** [comment]
Noticed that it should be root:root and it "worked" for a few minutes, now i have this issue instead:

jellyseerr-1  | 2024-08-29T01:07:42.670Z [[31merror[39m]: SyntaxError: Unexpected token { in JSON at position 8004
jellyseerr-1  |     at JSON.parse ()
jellyseerr-1  |     at Settings.load (/app/dist/lib/settings.js:334:61)
jellyseerr-1  |     at /app/dist/index.js:100:52 
jellyseerr-1  | error Command failed with exit code 1.
jellyseerr-1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
jellyseerr-1  | yarn run v1.22.19
jellyseerr-1  | $ NODE_ENV=production node dist/index.js

Not entirely sure how i managed to get that and it seams i have no option to repair it either?

**Reply by fallenbagel** [reply]
Check your settings.json file. It sounds like it's corrupted or broken.

I would suggest you hop onto discord if you want troubleshooting support as we carry out our support there

**Comment by mattish91** [comment]
For some reason it has fixed it self, had a power outage today and now everything seams to be fine. 🤔
